Documented the database vacuumn necessities. (#516)
digital-asset/docs.daml.com: docs/2.8.0/docs/canton/usermanual/upgrading.rst

allows to change the database schema. How long the migration takes depends on the version
of the binary (see migration notes), the size of the database and the performance of the database server.

> Before starting your node, we recommend that you clean-up your database. On Postgres, you would run

Reviewer: ```suggestion
We recommend cleaning up your database before you start your node. On Postgres, run
```

* The ``ContractKeyNotVisible`` error (previously encapsulated by ``GenericInterpretationError``) is now transformed into a ``ContractKeyNotFound`` to avoid information leaking.

Upgrade to Release 2.6
^^^^^^^^^^^^^^^^^^^^^^

> Version 2.6 will change the database schema used. Therefore, you will have to perform the

Reviewer: ```suggestion
Version 2.6 changes the database schema used. Therefore, you must perform the
```

> database migration steps. Please note that depending on the size of the database, this operation can take many hours. 

Reviewer: ```suggestion
database migration steps. Depending on the size of the database, this operation can take many hours. 
```

> Ensure that before starting your nodes, you vacuum your database to avoid long startup times. The participant 

Reviewer: ```suggestion
Vacuuming your database before starting your nodes helps avoid long startup times. Otherwise, the participant 
```

> node will refuse to startup due to initially extremely long database response times otherwise.

Reviewer: ```suggestion
node can refuse to start due to extremely long initial database response times.
```
